File: jdroid-java-core/src/test/java/com/jdroid/java/utils/StreamUtilsTest.java
package com.jdroid.java.utils;

import org.testng.annotations.Test;

import java.io.ByteArrayInputStream;
import java.io.InputStream;

import static org.testng.Assert.assertTrue;
import static org.testng.AssertJUnit.assertFalse;

public class StreamUtilsTest {

	@Test
	public void isEquals() {
		InputStream inputStream1 = new ByteArrayInputStream("123".getBytes());
		InputStream inputStream2 = new ByteArrayInputStream("123".getBytes());
		InputStream inputStream3 = new ByteArrayInputStream("124".getBytes());

		assertTrue(StreamUtils.isEquals(inputStream1, inputStream2));
		assertFalse(StreamUtils.isEquals(inputStream1, inputStream3));
	}

}

File: jdroid-java-core/src/test/java/com/jdroid/java/utils/StreamUtilsTest.kt
package com.jdroid.java.utils

import org.testng.Assert.assertTrue
import org.testng.AssertJUnit.assertFalse
import org.testng.annotations.Test
import java.io.ByteArrayInputStream

class StreamUtilsTest {

    @Test
    fun isEquals() {
        val inputStream1 = ByteArrayInputStream("123".toByteArray())
        val inputStream2 = ByteArrayInputStream("123".toByteArray())
        val inputStream3 = ByteArrayInputStream("124".toByteArray())

        assertTrue(StreamUtils.isEquals(inputStream1, inputStream2))
        assertFalse(StreamUtils.isEquals(inputStream1, inputStream3))
    }
}
